### ISSUE TITLE
Add helper ops for metrics arithmetics

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#414]
Following the discussion from [here](https://github.com/pytorch/ignite/pull/410#issuecomment-457105415)

> I was wondering whether it is possible to introduce a metric that performs mean, Mean, and compact the following piece of code:

```python
    precision = Precision(average=False)
    recall = Recall(average=False)
    F1 = precision * recall * 2 / (precision + recall + 1e-20)
    F1 = MetricsLambda(lambda t: torch.mean(t).item(), F1)
```
> to:
```python
    precision = Precision(average=False)
    recall = Recall(average=False)
    metrics = Mean(precision * recall * 2 / (precision + recall + 1e-20))
```

Idea is to provide some helper classes like `Mean`, `Sum` and others.

cc @amitibo
